metadata-service/factories/src/main/java/com/linkedin/gms/factory/search/SearchDocumentTransformerFactory.java

```diff
@@ -17,8 +17,11 @@ public class SearchDocumentTransformerFactory {
   @Value("${elasticsearch.index.maxObjectKeys}")
   private int maxObjectKeys;
 
+  @Value("${elasticsearch.index.maxFieldValue}")
+  private int maxFieldValue;
```

Comment:
maxValueLength?

Reply:
Reading this name it sounds like it presents a maximum number for something.. Not a length of a string